src/Dnd__DndManager.re

```diff
@@ -291,9 +291,18 @@ module Make = (Context: Context.T) => {
     };
   };
 
+  let invokeHook = (fn: option(unit => unit)) =>
+    switch (fn) {
+    | Some(fn) => fn()
+    | None => ()
+    };
+
   [@react.component]
   let make =
       (
+        ~onDragStart: option(unit => unit)=?,
```

Comment:
For my specific use case, I'd want to know which item is being dragged/dropped. Could these callbacks pass the `itemId` as an argument at a minimum? I could see the other data (`containerId`, etc.) being useful in some cases as well (for instance, if you wanted to update the styling of a container in response to an item being dragged out of or into it). I don't _need_ anything other than `itemId` myself, but if you had to add the other stuff later it'd be a breaking change, so maybe it's worth including it now?

Reply:
> Could these callbacks pass the itemId as an argument at a minimum?
Totally possible, will do.

> for instance, if you wanted to update the styling of a container in response to an item being dragged out of or into it
This wouldn't be useful in this callback for this use-case since it gets fired only on drag start but not when container gets changed. So it would require some special handler.

Let's proceed with the current set of handlers for now if it's enough for your current use cases. And add others as use-cases appear to figure proper api and try it out on the real world app before changing public interface.

Reply:
Ok, it's done. Please, take a look.

Reply:
👍 this looks great! Thanks so much!

Reply:
np! released in `1.1.0`